suseRegisterInfo/suseRegister/info.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2010-2014 Novell
+# Copyright (c) 2010-2021 SUSE LLC
```

Comment:
Not sure if you need to keep the old changelog? What are we doing at other places?

Reply:
SUSE was part of Novell at the time, I think the change is right. Most copyright headers from that time say "SUSE" not "Novell" and the authors were SUSE, not Novell.